donation-tracker/src/components/donationPill.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+import { Typography } from '@mui/material';
+
+import { DonationItem } from '../types/donationItem';
+
+const DonationPill = ({ donationItem }: { donationItem: DonationItem }) => {
+  return (
+    <Typography
+      fontSize="12px"
+      fontWeight="bold"
+      px={1}
+      py={0.5}
+      whiteSpace="nowrap"
+      sx={{ backgroundColor: donationItem.remainingNeed ? 'rgba(25,118,210,0.1)' : 'rgba(0,255,0,0.25)' }}
+      color={donationItem.remainingNeed ? 'rgba(25,118,210,0.75)' : 'rgba(0,164,0,0.75)'}
+      border={`2px solid ${donationItem.remainingNeed ? 'rgba(25,118,210,0.75)' : 'rgba(0,164,0,0.75)'}`}
+      borderRadius={5}
+      textTransform="uppercase"
+      component="div"
+    >
+      {donationItem.remainingNeed ? `+ ${donationItem.remainingNeed} ${donationItem.unit} needed` : 'Goal accomplished'}
```

Comment:
Are you planning to make these strings configurable (for DE) - or is this out of focus?

Reply:
It looks horrible, if it is showing both languages at once. Unless we have proper i18n I'd recommend to stay with short and crisp English terms.